### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by jessgusclark**
Updating security document at the request of @bcodesido to match the other Collective Security disclosures.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,143 +1,27 @@
...
+# Reporting Security Issues
+
+The RootstockCollective team and community take security vulnerabilities very seriously. Although this project is currently in its bootstrapping phase (MVP) and falls outside the scope of a Bug Bounty Program, we sincerely appreciate your responsible disclosure and will make every effort to acknowledge your contributions.
 
-We are committed to conduct our security process in a professional and civil manner. Public shaming, under-reporting or misrepresentation of vulnerabilities will not be tolerated.
 
 ## Responsible Disclosure
 
-For all security related issues, RIF Wallet has two main points of contact. Reach us at <security@rootstocklabs.com> or refer to our [Bug Bounty Program](https://www.rootstocklabs.com/bug-bounty-program). **Do not open up a GitHub issue if the bug is a security vulnerability**
+For all security related issues, use the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/RootstockCollective/dao-frontend/security/advisories/new) tab.
```
Comment (bernacodesido) [inline SECURITY.md]: repo name is wrong

File: SECURITY.md [hunk 2]
```diff
@@ -1,143 +1,27 @@
...
+
+The RootstockCollective team will send a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
 
-**Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/rsksmart/rif-wallet-services/issues).
+**Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/RootstockCollective/dao-frontend/issues).
```
Comment (bernacodesido) [inline SECURITY.md]: same as above

**Review by bernacodesido — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (sleyter93) [reply SECURITY.md]: The repo name is updated

File: SECURITY.md (hunk 2 above)
Comment (sleyter93) [reply SECURITY.md]: The repo name is updated

**Review by bernacodesido — APPROVED** [review]

**Pull request merged**